linalg/hypre.hpp

```diff
@@ -638,6 +642,9 @@ class HypreSmoother : public Solver
    /// Combined coefficients for windowing and Chebyshev polynomials.
    double* fir_coeffs;
 
+   /// Indicate what MultTranspose would do when it is called.
+   TransposeAction transpose_action;
+
 public:
    /** Hypre smoother types:
```

Comment:
I suggest having the constructor(s) set the `transpose_action` flag if we know it's symmetric (Jacobi and l1-jacobi for sure, maybe some others.)

Reply:
Good suggestion. In that case, we should also set the flag in `HypreSmoother::SetType( ... )`.